acceptance/lib/helper.rb

```diff
@@ -28,7 +28,7 @@ def self.initialize_config(options)
 
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Development Build Version",
-                         "PUPPET_BUILD_VERSION", "3.7.3")
+                         "PUPPET_BUILD_VERSION", "4.0.0")
```

Comment:
this probably isn't the right build version, since I don't think puppet-agent is going to be 4.0.0, but I'm not sure what the right default is.